pkg/csi/service/wcpguest/controller.go

```diff
@@ -361,7 +361,7 @@ func (c *controller) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequ
 				VolumeContext: attributes,
 			},
 		}
-		if commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.TKGsHA) &&
```

Comment:
Can we combine this with if block at :337 and move the CreateVolumeResponse to line :336?